samples/csharp_dotnetcore/81.skills-skilldialog/DialogSkillBot/DialogSkillBot.csproj

```diff
@@ -9,10 +9,10 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.AI.Luis" Version="4.8.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.Dialogs" Version="4.8.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.8.0" />
-    <PackageReference Include="Microsoft.Recognizers.Text.DataTypes.TimexExpression" Version="1.1.5" />
+    <PackageReference Include="Microsoft.Bot.Builder.AI.Luis" Version="4.9.0" />
+    <PackageReference Include="Microsoft.Bot.Builder.Dialogs" Version="4.9.0" />
+    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.9.0" />
+    <PackageReference Include="Microsoft.Recognizers.Text.DataTypes.TimexExpression" Version="1.3.2" />
   </ItemGroup>
```

Comment:
This explicit dependency can be removed, DataTypes.TimexExpression is referenced by AdaptiveExpressions that is referenced by AI.Luis.

Reply:
How about we do that with a separate commit?
(there are a few other follow ups - like new warning messages 😢)

Reply:
OK looking more closely this is correct. Core bot uses timex expressions directly.
Looks like the dependency on adaptiveexpressions is accidental